components/preparing_workspace/use_case.tsx

```diff
@@ -0,0 +1,132 @@
+                                    transform: 'translateY(-100%)',
+                                    top: '-20px',
+                                }}
+                                noLeft={true}
+                            />
+                            <LaptopSVG/>
+                            <PageLine
+                                noLeft={true}
+                            />
+                        </div>
+                        {props.previous}
+                        <Title>
+                            <FormattedMessage
+                                id={'onboarding_wizard.use_case.title'}
+                                defaultMessage='How do you plan to use Mattermost?'
+                            />
+                        </Title>
+                        <Description>
+                            <FormattedMessage
+                                id={'onboarding_wizard.use_case.description'}
+                                defaultMessage="This will help us set up your workspace in a way that's most relevant to you. Select all that apply."
+                            />
+                        </Description>
+                        <PageBody>
+                            <MultiSelectCards
+                                next={props.next}
+                                cards={[
+                                    {
+                                        onClick: () => props.setOption('channels'),
+                                        icon: <ChannelsSVG/>,
+                                        id: t('onboarding_wizard.use_case.channels'),
+                                        defaultMessage: 'Team communication and collaboration',
+                                        checked: props.options.channels,
+                                    },
+                                    {
+                                        onClick: () => props.setOption('boards'),
+                                        icon: <BoardsSVG/>,
+                                        id: t('onboarding_wizard.use_case.boards'),
+                                        defaultMessage: 'Project planning and management',
+                                        checked: props.options.boards,
+                                    },
+                                    {
+                                        onClick: () => props.setOption('playbooks'),
+                                        icon: <PlaybooksSVG/>,
+                                        id: t('onboarding_wizard.use_case.playbooks'),
+                                        defaultMessage: 'Processes, workflows, and automation',
+                                        checked: props.options.playbooks,
+                                    },
+                                ]}
+                            />
+                        </PageBody>
+                        <div>
+                            <button
```

Comment:
I'm wondering if there's a way to extract the buttons to make it a reusable component over the pages, but not sure it's worth it